lib-android/src/test/java/com/devinci/lib/AttributeSetUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.devinci.lib;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.util.AttributeSet;
+import org.robolectric.res.Attribute;
+import org.robolectric.shadows.RoboAttributeSet;
+
+import static org.assertj.core.util.Lists.newArrayList;
+import static org.assertj.core.util.Preconditions.checkNotNull;
+import static org.robolectric.Shadows.shadowOf;
+
+public final class AttributeSetUtils {
```

Comment:
We should think about `lib-android-test-robolectric` module. This class  may be useful in other apps.